_tests/browser_tests/page_objects/page_careers-students-and-graduates.js

```diff
@@ -0,0 +1,10 @@
+function careers_studentsAndGraduates() {
```

Comment:
I'm happy to change the name to something else. Let's chat about how we want to handle the naming for subpages.